app/views/movies/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Welcome <%= @user.email %>!</h1>
+
+<section id="top_movies">
```

Comment:
I think we may have to have conditional to parse out search vs the top movies...